src/components/Inputs/TextMaskedInput/TextMaskedInput.js

```diff
@@ -17,22 +17,18 @@ export const TextMaskedInput = (props) => {
     allCaps,
     validator,
     formChangeHandler,
+    initialValue,
+    doValidation,
     ...restProps
   } = props
 
   const [getError, setError, validate] = useErrorMessage(validator)
-  const [value, setValue] = useState('')
-  const [touched, setTouched] = useState(false)
-  const [doValidation] = useInputValidation({validate, setError, formChangeHandler})
+  const [value, setValue] = useState(initialValue || '')
+  const [touched, setTouched] = useState(initialValue ? true : false)
+  const [internalDoValidation] = useInputValidation({validate, setError, formChangeHandler})
```

Comment:
Explanation:
TextMaskedInput can take care of validation for you: use case you want a custom last 4 SSN and you want to make this declaratively (see latest Form.md example), or, you may want to do validation yourself but leverage it's text-mask e.g. ZipInput today